### PR TITLE
Implemented a MVP version of biometric authentication.

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

[comment]: <> (Description of the feature or fix)
[comment]: <> (Include link to Figma designs, if relevant)
[comment]: <> (Include link to Notion ticket, if relevant)

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

## Screenshots

[comment]: <> (Please attach screenshots showing the change, if relevant)

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Checklist (for merger)

- [ ] Make sure the right merge strategy is chosen, commit merge into master and squash merge into develop.

### CODE REVIEW

File: src/features/auth/BiometricAuthScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      const results = await LocalAuthentication.authenticateAsync();
+
+      if (results.success) {
+        setResult(EResult.SUCCESS);
+      } else if (results.error === 'unknown') {
+        // Getting this response means the device doesn't allow biometric authentication anymore.
+        // This mostly happens after the user tried too many times.
+        setResult(EResult.DISABLED);
+      } else if (
+        results.error !== 'user_cancel' &&
```
Comment (juliatan) [inline src/features/auth/BiometricAuthScreen.tsx:52]: Question: should these 3 lines say `results.error === 'user_cancel' || results.error === 'system_cancel' || results.error === 'app_cancel'`...

Comment (aikewoody) [reply src/features/auth/BiometricAuthScreen.tsx:52]: Yes indeed! That's the pitfall of writing code without running it 🙃

Comment (juliatan) [reply src/features/auth/BiometricAuthScreen.tsx:52]: Cool, I'll amend and push. 😄

File: ios/Covid.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -698,7 +698,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.joinzoe.covid-zoe.qa";
 				PRODUCT_NAME = "covid-zoe Stage";
-				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc com.joinzoe.covid-zoe.qa";
+				PROVISIONING_PROFILE_SPECIFIER = "match Development com.joinzoe.covid-zoe.qa";
```
Comment (aikewoody) [inline ios/Covid.xcodeproj/project.pbxproj:701]: These changes shouldn't be checked into git

**Comment by aikewoody** [comment]
@juliatan Thanks for getting this working in our native app! There are a few changes we need to revert but other than that this PR looks great

File: ios/Covid.xcodeproj/project.pbxproj (hunk 2 above)
Comment (juliatan) [reply ios/Covid.xcodeproj/project.pbxproj:701]: Ah I wasn't sure... so do you think we should add to .gitignore?
I think if you don't change these settings, your local build in XCode might fail...

**Pull request merged**